Student-Submissions/script.js

```diff
@@ -0,0 +1,76 @@
+const submissions = [
+  { name: "Jane", score: 95, passed: true },
+  { name: "Joe", score: 77, passed: true },
+  { name: "Jack", score: 59, passed: false },
+  { name: "Jill", score: 88, passed: true },
+];
+
+const addSubmissionfunction = (array, newName, newScore) => {
+  const newPassed = newScore >= 60 ? true : false;
+  array.push({ name: newName, score: newScore, passed: newPassed });
+};
+
+const deleteSubmissionByIndex = (array, index) => array.splice(index, 1);
+
+const deleteSubmissionByNamefunction = (array, name) => {
+  const index = array.findIndex((submission) => submission.name === name);
+  if (index !== -1) {
+    array.splice(index, 1);
+  }
+};
+
+const editSubmission = (array, index, score) => {
+  array[index].score = score;
+  array[index].passed = score >= 60 ? true : false;
+};
+
+const findSubmissionByName = (array, name) =>
+  array.find((submission) => submission.name === name);
+
+const findLowestScore = (array) => {
+  const lowestSubmission = array[0];
```

Comment:
Should declare with 'let' here. We are potentially reassigning the value on line 34, so we don't want to use const.